client/container_resize_test.go

```diff
@@ -0,0 +1,52 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+)
+
+func TestContainerResizeError(t *testing.T) {
+	client := &APIClient{
+		HTTPCli: newMockClient(errorMockResponse(http.StatusInternalServerError, "Server error")),
+	}
+	err := client.ContainerResize(context.Background(), "nothing", "", "")
+	if err == nil || !strings.Contains(err.Error(), "Server error") {
+		t.Fatalf("expected a Server Error, got %v", err)
+	}
+}
+
+func TestContainerResize(t *testing.T) {
+	expectedURL := "/containers/container_id/resize"
+
+	httpClient := newMockClient(func(req *http.Request) (*http.Response, error) {
+		if !strings.HasPrefix(req.URL.Path, expectedURL) {
+			return nil, fmt.Errorf("Expected URL '%s', got '%s'", expectedURL, req.URL)
```

Comment:
According to the [code style rules of pouch](https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md), No matter log or error, first letter of the message must be lower-case

Reply:
I am following the example write by allencloud in issue #966 .Is there special?

Reply:
I am afraid that @zhuangqh has made a point and we should change all the case problem in mock test. And I have noticed that the same problem exists in other part of Pouch. WDYT? @allencloud

Reply:
Yes, I think what @zhuangqh mentioned is reasonable. Could someone fix this totally in a single pull request. @ZouRui89

Reply:
@Dewey-Ding Could you fix the case problem? Thx a lot.

Reply:
maybe i can do it in this pr.

Reply:
@allencloud @ZouRui89